src/include/migraphx/process.hpp

```diff
@@ -49,9 +54,12 @@ struct MIGRAPHX_EXPORT process
     ~process() noexcept;
 
     process& cwd(const fs::path& p);
+    process& env(const std::vector<std::string>& envs);
+    process& launcher(const fs::path& launcher);
```

Comment:
What is this? If we are using a launcher like `ccache` then we will set `cmd` to `ccache` and then pass the executable as an argument.